### ISSUE TITLE
Eliminating "_DATA_TABLE" from names of .csv files

### ISSUE THREAD

**Issue by pszolovits** [MIT-LCP/mimic-code#43]
The current naming convention of calling the .csv files things like ADMISSIONS_DATA_TABLE.csv seems redundant, and also causes extra work to edit the output of my heuristic structure finder (https://github.com/mitmedg/csv2mysql), which thus generates scripts like
CREATE TABLE ADMISSIONS_DATA_TABLE ...

Why not just change them all by eliminating _DATA_TABLE from the name of the file?

**Comment by tompollard** [comment]
I was thinking the same thing recently when creating the bash script for our data export (login needed - https://github.com/MIT-LCP/mimic-iii-building/tree/master/data_export).

The "_DATA_TABLE" is added by default when dumping CSV files using the Oracle GUI, but as we now export the data via custom scripts there's no reason to include it.

We'll go ahead with this change sometime soonish...

**Comment by tompollard** [comment]
Rename csvs with:

``` bash
rename 's/_DATA_TABLE*//' *.csv
```

**Comment by tompollard** [comment]
CSV files and checksums have been updated on PhysioNet and testing servers.

**Comment by tompollard** [comment]
"_DATA_TABLE" has been removed from the filename in the data and associated scripts. Closing issue...

**Comment by alistairewj** [comment]
Can do this in Windows powershell as follows:

```
get-childItem *_DATA_TABLE.csv | rename-item -newname { $_.name -replace '_DATA_TABLE','' }
```

**Comment by tompollard** [comment]
Closed issue Alistair, closed issue. If _DATA_TABLE is in the filename, it's time to upgrade to the current version!

**Comment by alistairewj** [comment]
Just downloaded ADMISSIONS and CALLOUT from Physionet and while the .csv.gz
don't have DATA_TABLE in the name, when I extract them the final .csv does.

**Comment by tompollard** [comment]
Oh, what...I'll sort that out.

**Comment by tompollard** [comment]
I just downloaded the ADMISSIONS and CALLOUT tables (+ a couple of others) from Physionet and decompressed using gzip -d *.gz. The decompressed files do not contain  DATA_TABLE in the name:

-rw-r--r--  1 name  group  12738420 Jan 24 10:58 ADMISSIONS.csv
-rw-r--r--  1 name  group   6397862 Jan 24 10:58 CALLOUT.csv
-rw-r--r--  1 name  group     43532 Jan 24 10:56 D_LABITEMS.csv

Something weird going on here. Possible that you're somehow downloading an older version? Or some form of caching related to the two PhysioNet servers?

**Comment by alistairewj** [comment]
I used 7zip on Windows 7. Not sure if that changes things..

**Comment by alistairewj** [comment]
https://physionet.org/works/MIMICIIIClinicalDatabase/files/version_1_3/ADMISSIONS.csv.gz

Opening this on 7zip:

![image](https://cloud.githubusercontent.com/assets/1282676/12537338/9eabccdc-c28a-11e5-958a-7560e8187763.png)

**Comment by tompollard** [comment]
Oh, that's annoying, I see what's happened. I'll update the export scripts and we can include the new filename in the next release.

**Comment by tompollard** [comment]
https://github.com/MIT-LCP/mimic-iii-building/commit/496ae0db006dae6427c8c6f726c4b62e7cb03767

**Comment by alistairewj** [comment]
Fixed with release of v1.4.
